content/src/storage/RepositoryFactory.ts

```diff
@@ -23,6 +25,7 @@ export class RepositoryFactory {
       }
     }
 
-    return build(connection, contentCredentials, rootCredentials)
+    const database = await build(connection, contentCredentials, rootCredentials)
+    return new Repository(database, new RepositoryQueue())
```

Comment:
can we add two env variables to configure the queue in case we need to override the defaults?